src/components/Calculator.js

```diff
@@ -1,17 +1,18 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import './style.css';
 import CalculatorTitle from './AppTitle';
 import calculate from '../logic/calculate';
 
-function Calculator() {
-  const [display, setdisplay] = useState({
+// eslint-disable-next-line react/prop-types
+function Calculator({ initialState }) {
```

Comment:
- [ ] Good job making the changes as requested by the previous reviewer @MasumaJaffery, but please Consider removing disabled liters. Remember linter is here to force you to write clean code and help you identify what could possibly be improved on the current codebase.